src/v2/Apps/Order/Routes/Payment/index.tsx

```diff
@@ -208,9 +257,11 @@ export const PaymentRoute: FC<Props> = props => {
               contextPageOwnerType={OwnerType.ordersPayment}
             />
             <Spacer mb={[2, 4]} />
-            <Media at="xs">
-              <ContinueButton onClick={onContinue} loading={isLoading} />
-            </Media>
+            {selectedPaymentMethod !== "US_BANK_ACCOUNT" && (
```

Comment:
**suggestion (non-blocking):** We don't want to show the Continue button when the user has selected US Bank Account because we hand off control to Stripe for the next steps, correct? If so, consider abstracting this logic into a method named something like `isPaymentMethodAddedWithStripeUI`.

Reply:
Added the condition just because on the page were 2 buttons